schema-generic.graphql

```diff
@@ -303,6 +339,15 @@ type Pool @entity {
   " Per-block reward token emission as of the current block normalized to a day, in USD value. This should be ideally calculated as the theoretical rate instead of the realized amount. "
   rewardTokenEmissionsUSD: [BigDecimal!]
 
+  " Day ID of the most recent daily snapshot "
+  lastSnapshotDayID: Int!
+
+  " Hour ID of the most recent hourly snapshot "
+  lastSnapshotHourID: Int!
+
+  " Timestamp of the last time this entity was updated "
+  lastUpdateTimestamp: BigInt!
+
```

Comment:
We could prepend the `-`. I did not do it, because usually those we use for supplementary fields. However, I think we want this to be a field in the schema for all subgraphs, since it should be used commonly. I'd imagine if a protocol is implementing the new schema, we'd also want them to use the new methodology for snapshots, meanly these will always be used, no? I made them required, because if we want to standardize the snapshotting mechanism, I thought this would encourage it. Also, I thought it might potentially be interesting for users to see these fields on the main entity

Reply:
Hmm 🤔 

The encouraging part, to keep snapshot methods consistent, sells me. I would have said that having them immutable already achieves that, but this might help also keep the same intervals and stuff. Sounds good 👍